Procfile

```diff
@@ -1 +0,0 @@
-web: http-server -p $PORT
```

Comment:
Why has this been deleted? It's used by Heroku

Reply:
(The $PORT bit is needed)

Reply:
Nope!

* The default process is `web`
* `http-server` uses the $PORT env if no port is declared.
* Heroku automatically runs `rpm start`, which runs `http-server`

Reply:
Simplifying the npm scripts is separate from the goals of this PR. Have reverted this particular change and will open a separate PR.